pkg/operator/ceph/object/objectstore_test.go

```diff
@@ -211,7 +211,7 @@ func deleteStore(t *testing.T, name string, existingStores string, expectedDelet
 	}
 	err = deleteRealmAndPools(context, spec)
 	assert.Nil(t, err)
-	expectedPoolsDeleted = 6
+	expectedPoolsDeleted = 7
```

Comment:
What is causing an extra pool to be deleted? Did we miss a pool before?

Reply:
I can new pool named <>.rgw.otp in newer ceph versions, see I have added it in https://github.com/rook/rook/pull/10728/files#diff-3d31541d76db0087520217d3ccb2b5d704b3a7e05a31167c80bcf69b2cfcaec8R64